apps/web/components/Organization/Wrapper/Themes/VinnueftirlitidTheme/VinnueftirlitidHeader.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+import cn from 'classnames'
+
+import { DefaultHeader } from '@island.is/web/components'
+import { OrganizationPage } from '@island.is/web/graphql/schema'
+import { useLinkResolver } from '@island.is/web/hooks'
+
+import * as styles from './VinnueftirlitidHeader.css'
+
+interface HeaderProps {
+  organizationPage: OrganizationPage
+}
+
+const VinnueftilitidHeader: React.FC<React.PropsWithChildren<HeaderProps>> = ({
+  organizationPage,
+}) => {
+  const { linkResolver } = useLinkResolver()
+  const themeProp = organizationPage.themeProperties
+  return (
+    <div
+      className={cn({
+        [`${styles.headerBorder}`]: true,
+        [`${styles.headerBorderWidth}`]: !themeProp.fullWidth,
+      })}
+    >
+      <DefaultHeader
+        title={organizationPage.title}
+        image="https://images.ctfassets.net/8k0h54kbe6bj/6OqSuq1pVpiOxZ5o6fzGSK/27622b5aeb9d7ddbca53a1cc3d203b35/vinnueftirlitid.png"
+        imagePadding={themeProp.imagePadding ?? '0'}
+        background=" repeat url('https://images.ctfassets.net/8k0h54kbe6bj/2MLg9apOlM56iVrrs9Gnn0/5a0085da93fa7a532d2388e75f77522b/VER-bg-banner.svg')"
```

Comment:
Perhaps reading this url from a namespace might be useful if we'd like to later on change the image and then we don't have to touch the code 🙃